### DISCUSSION TITLE
Welcome to DataBot Discussions!

### DISCUSSION THREAD

**Discussion by vvaezian** [Intellimenta/DataBot#1] [Announcements]
## Welcome!
 Here you can ask questions and find answers about DataBot. 
To get started click on the "New discussion" button on the previous page.

**Comment by theRMH** [comment]
Custom connecter will be a game changer for digital marketing agencies, especially with Google sheet

**Reply by vvaezian** [reply]
Hi @theRMH, thanks for the feedback. For Google sheet, a workaround is to connect it to BigQuery, and use the BigQuery database connection in DataBot.
